pkg/kapp/resources/association_label.go

```diff
@@ -12,7 +12,6 @@ import (
 
 const (
 	kappAssociationLabelKey = "kapp.k14s.io/association"
-	kappAssociationLabelV1  = "v1"
```

Comment:
this one is actually looks like should be used in v1Value() of AssociationLabel. it's currently using kappIdentityAnnV1 which has a same value but technically isnt for this class.

Reply:
Added back and now using `kappAssociationLabelV1`